project_stage_state/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Project Stage State
+===================
+
+This module restores the `state` fields to Project Stages, removed in Odoo 8.0.
+
+For some use cases it‘s necessary to be able to map the multiple Stages into
+a few broad groups.
+
+For example, this can allow to define automated actions and business logic for
+Tasks not yet “Started”, knowing that “Started” means different Stages in
+different Projects.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Project -> Configuration -> Stages and click on a stage
+#. Select the state you would like to associate that stage with from the dropdown "State" menu
+#. Save your changes
+#. Go to Project -> Dashboard and click on a project
+#. Click on task in the stage you just edited
+#. Under the "Extra Info" tab, you can see the "State" field for that task
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/12.0
+
+Known issues / Roadmap
```

Comment:
@anandkansagra If not use then remove it

Reply:
Thanks, Done